.github/workflows/review.yml

```
@@ -8,55 +8,229 @@ name: r3-review
+    # coderabbit, which is an `external` backend this tool never executes, and
+    # the run would end where it began.
+    if: >-
+      github.event.pull_request.head.repo.full_name == github.repository
+      && vars.MF_R3_ENDPOINT != ''
+    env:
+      # The machine layer, materialised on the runner. docs/adr/0006 splits
+      # policy from reachability: .framework.toml names the reviewers this
+      # repository wants and never how any of them is reached, so CI supplies
+      # the second half here rather than by committing an endpoint. Pointing
+      # MF_CONFIG_HOME at the runner's own temp directory keeps it to this job.
+      MF_CONFIG_HOME: ${{ runner.temp }}/mf-config
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)